### ISSUE TITLE
Disable Kubeflow fails

### ISSUE THREAD

**Issue by chris-sanders** [ubuntu/microk8s#2296]
[inspection-report-20210527_154312.tar.gz](https://github.com/ubuntu/microk8s/files/6555265/inspection-report-20210527_154312.tar.gz)

Trying to disable Kubeflow after a failed install results in a syntax error in the disable script.

```
ubuntu@runner-7v6kiq:~$ microk8s disable kubeflow
  File "/snap/microk8s/2213/scripts/wrappers/common/../../../actions/disable.kubeflow.sh", line 52
    click.echo(f"Destroying Kubeflow {resource}...")

```

**Comment by ktsakalozos** [comment]
I believe this is fixed on edge and should be moved to stable with the next patch release. Right @DomFleischmann ?

Related issue https://github.com/ubuntu/microk8s/issues/2263

**Comment by DomFleischmann** [comment]
Correct, this should be fixed on 1.20 edge.
